routes/index.js

```diff
@@ -1,18 +1,16 @@
-var express = require('express');
-var router = express.Router();
+import express from 'express';
+const router = express.Router();
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).